packages/admin/admin/src/common/breadcrumbs/Breadcrumbs.tsx

```diff
@@ -0,0 +1,116 @@
+import { ChevronRight } from "@comet/admin-icons";
+import { type ComponentsOverrides, Typography } from "@mui/material";
+import { css, type Theme, useThemeProps } from "@mui/material/styles";
+import { Fragment, type ReactNode } from "react";
+
+import { createComponentSlot } from "../../helpers/createComponentSlot";
+import { type ThemedComponentBaseProps } from "../../helpers/ThemedComponentBaseProps";
+
+type BreadcrumbsClassKey = "root" | "breadcrumbsList" | "breadcrumbsItem" | "separator";
+
+export interface BreadcrumbsItemProps {
+    url: string;
+    title: ReactNode;
+}
+
+interface BreadcrumbsProps
+    extends ThemedComponentBaseProps<{
+        root: "div";
+        breadcrumbsList: "div";
+        breadcrumbsItem: typeof Typography;
+        separator: typeof ChevronRight;
+    }> {
+    items: BreadcrumbsItemProps[];
+}
+
+const Root = createComponentSlot("div")<BreadcrumbsClassKey>({
+    componentName: "Breadcrumbs",
+    slotName: "root",
+})();
```

Comment:
Is this unstyled div necessary? Could `BreadcrumbsList` simply be the "Root" slot?